### PR TITLE
lock screen widgets

### PR DESCRIPTION
**Pull request by brindy**
Task/Issue URL: https://app.asana.com/0/414709148257752/1204600823062806/f
Tech Design URL:
CC:

**Description**:

Adds some initial Lock Screen widgets to the app.

* Favorites - launches a new tab showing the home page
* Fire Button - launches the app and runs the fire button
* Email Protection - if signed in creates an email address and puts it in on the pasteboard, otherwise launches the email protection web page
* Search - launches a new tab and focuses for a search
* Voice - launches a new tab using voice search

Refactored app deep links into an enum.

Removed default browser pixel as not used at this point.

Refactored email manager to be in the main view controller.

**Steps to test this PR**:
1. Lock the device and customise the screen.
2. Add all the widgets and test them.
3. Check the copy matches.
4. Check about screen link still works.
5. Check email protection (sign in, sign out, autofill, etc)

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [ ] iOS 16

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Comment by samsymons** [comment]
@brindy Is this ready for review now, or still a draft? (Just checking if this needs to be assigned to someone for review)

**Comment by brindy** [comment]
Thanks @samsymons - still draft.  Waiting on Copy, then ship review. I thought you could convert back to draft, but I can't see how so just renamed it.

File: Widgets/DeepLinks.swift
Comment (samsymons) [inline Widgets/DeepLinks.swift:1]: I notice that this file also exists in the DuckDuckGo app target but isn't being used, we can probably remove it.

**Review by samsymons — APPROVED** [review]
LGTM - I tested all of the new widget types, all behaving as expected. Also tested other areas where we're using app deep links, such as the About option in settings. All seems to be working as expected, nicely done!

File: Widgets/DeepLinks.swift
Comment (brindy) [reply Widgets/DeepLinks.swift:1]: Thanks. Probably never was.

**Comment by brindy** [comment]
@samsymons had to move things around in order to use the email manager.  Can you give it another go over and also check that email protection works (sign in, sign out, autofill, etc).  Thanks!

**Review by samsymons — APPROVED** [review]
New changes look good! Checked the new icons, and tested the various pieces of functionality that you pointed out (Email, Autofill, etc.). No issues that I could see, LGTM!

**Pull request merged**